docs/scarpet/resources/editors/npp/scarpet_v1.7_notepad++.xml

```diff
@@ -60,6 +60,8 @@
 
                 scoreboard scoreboard_add scoreboard_remove scoreboard_display
 
+                team_add team_list team_remove team_leave team_property
```

Comment:
this is technically not need - its 1.7 whatever that was - its just for posterity.